extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -114,11 +141,56 @@ export function VideoWrapper({
     };
   }, [mediasession, playing, metadata, play, pause]);
 
+  useImperativeHandle(
+    ref,
+    () => ({
+      // Standard HTMLMediaElement/Element
+      play,
+      pause,
+      requestFullscreen,
+      get currentTime() {
+        return playerRef.current.currentTime;
+      },
+      get duration() {
+        return playerRef.current.duration;
+      },
+      get autoplay() {
+        return autoplay;
+      },
+      get controls() {
+        return controls;
+      },
+      get loop() {
+        return loop;
+      },
+
+      // Non-standard
+      userInteracted,
+      mute: () => setMuted(true),
+      unmute: () => {
+        if (hasUserInteracted) {
+          setMuted(false);
+        }
+      },
+    }),
+    [
+      play,
+      pause,
+      requestFullscreen,
+      userInteracted,
+      hasUserInteracted,
+      autoplay,
+      controls,
+      loop,
+    ]
+  );
+
   return (
     <ContainWrapper
       contentRef={wrapperRef}
       className={className}
       style={style}
+      containsize={containsize}
```

Comment:
Hmm. I don't think there's such a prop.

Reply:
I think it's actually `containSize`, comes from `PreactBaseElement` and ends up set on the `<video>`.

https://github.com/ampproject/amphtml/blob/f40404f24df14ac49d71dfe375ef023a34ac4063/src/preact/base-element.js#L624-L632

Should I pass that to `ContainWrapper`?

Reply:
I see. I just forgot to clean it up. Ignore it for now and I'll send a separate PR.